sdk/storage/Azure.Storage.Files.Shares/src/autorest.md

```diff
@@ -4,7 +4,7 @@ Run `dotnet build /t:GenerateCode` to generate code.
 
 ``` yaml
 input-file:
-    - https://raw.githubusercontent.com/Azure/azure-rest-api-specs/d18a495685ccec837b72891b4deea017f62e8190/specification/storage/data-plane/Microsoft.FileStorage/stable/2025-05-05/file.json
+    - https://raw.githubusercontent.com/nickliu-msft/azure-rest-api-specs/17e085c8f2fabb718b69e44dd42ad6ac9cbedbcf/specification/storage/data-plane/Microsoft.FileStorage/stable/2025-11-05/file.json
```

Comment:
We need to merge the swagger PR for this feature and change this line to point at the merged swagger before we merged this .NET PR.